eng/tools/rush-runner.js

```diff
@@ -59,7 +59,11 @@ const spawnNode = (cwd, ...args) => {
   console.log(`Executing: "node ${args.join(' ')}" in ${cwd}\n\n`);
   const proc = spawnSync('node', args, { cwd, stdio: 'inherit' });
   console.log(`\n\nNode process exited with code ${proc.status}`);
-}
+
+  if (proc.status !== 0) {
+    process.exit(proc.status);
```

Comment:
I think `spawnNode()` should return `proc.status` and let the caller decide what to do

Reply:
Actually I think it should just set `process.exitCode`:

```js
if (proc.status !== 0) {
    if (proc.status === null) {
        // proc.status will be null if the subprocess terminated due to a signal, which I don't think
        // should ever happen, but if it does it's safer to fail.
        process.exitCode = 1;
    }
    else {
        process.exitCode = proc.status;
    }
}

This should also handle the "lint" looping case.  Lint should be run on all packages even if some fail, but the process exit code will be set to the last failure, which I think is the best behavior.
```